### DISCUSSION TITLE
Need udpates on ubuntu-24.04 LTS version

### DISCUSSION THREAD

**Discussion by srini-12345** [actions/virtual-environments#10486] [General questions] [answered]
I would like to update my Ubuntu servers, may I know when can we expect ubuntu-24.04 LTS version please?

**Comment by erik-bershel** [comment, marked as answer]
It's already available:
- https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md
- https://github.com/actions/runner-images/issues/9848

**Reply by srini-12345** [reply]
Hi Erik,  As per below link I still can see 24.04 showing as Beta version,
https://github.com/actions/runner-images/tree/main
![image](https://github.com/user-attachments/assets/10339999-1bab-4325-8e1a-96b5908745fb)

**Reply by erik-bershel** [reply]
The purpose of a `Beta` is to collect feedback on an image before it is released to GA. The goal of a `Beta` is to identify and fix any potential issues that exist on that image. It's a part of GitHub image lifecycle.
